### DISCUSSION TITLE
problem building an application after update: Cannot find package 'tslib' imported from /app/server/entries/pages/__layout.svelte.js

### DISCUSSION THREAD

**Discussion by christophemacabiau** [sveltejs/kit#4271] [Q&A] [answered]
### Describe the bug

I have a problem building an application after kit update 1.0.0-next.181 --> 1.0.0-next.294.

The production build is different, and I don't know why.
When I run my application I get the error `Error [ERR_MODULE_NOT_FOUND]: Cannot find package 'tslib' imported from /app/server/entries/pages/__layout.svelte.js`

### Reproduction
https://stackblitz.com/edit/sveltejs-kit-template-default-jpz1zt?file=build/server/entries/pages/index.svelte.js
run `npm run build`
and then open `build/server/entries/pages/index.svelte.js`
tslib is imported:
```js
import { n as noop, a as safe_not_equal, b as subscribe, r as run_all, i as is_function, c as create_ssr_component } from "../../chunks/index-110612cb.js";
import { __assign, __extends, __spreadArray } from "tslib";
```
I dont know if it is something wanted, but now it seems that tslib is no longer bundled.
The problem appears when I use the svelte-i18n package

### System Info

```shell
System:
    OS: Linux 5.11 Ubuntu 21.04 (Hirsute Hippo)
    CPU: (16) x64 AMD Ryzen 7 4800H with Radeon Graphics
    Memory: 998.07 MB / 30.78 GB
    Container: Yes
    Shell: 3.1.2 - /usr/bin/fish
  Binaries:
    Node: 16.10.0 - ~/.volta/tools/image/node/16.10.0/bin/node
    npm: 7.24.0 - ~/.volta/tools/image/node/16.10.0/bin/npm
  Browsers:
    Chrome: 99.0.4844.51
    Firefox: 96.0
  npmPackages:
    @sveltejs/adapter-node: ^1.0.0-next.71 => 1.0.0-next.71 
    @sveltejs/kit: 1.0.0-next.294 => 1.0.0-next.294 
    svelte: ^3.42.6 => 3.46.4
```

### Severity

blocking an upgrade

**Comment by Conduitry** [comment]
Yes, a reproduction is necessary.

**Comment by ITwrx** [comment]
@christophemacabiau it might be advisable to use the repo Discussions forum until you have confirmed it's actually a bug. That's what i'm trying to do. Fundamental things are still in flux, so asking for help there, instead, allows the devs to keep working on Sveltekit, without us slowing them down. :)

**Comment by christophemacabiau** [comment]
I added a reproduction. You can run `npm run build`.
If it's better for you, I can close this bug and go forward with a discussion as mentionned @ITwrx

**Comment by christophemacabiau** [comment]
I have a better (minimal) reproduction (see the description): I dont know if it is something wanted, but now it seems that tslib is no longer bundled. The problem appears when I use the svelte-i18n package.
The build directory is no longer self-sufficient?

**Comment by christophemacabiau** [comment, marked as answer]
https://github.com/sveltejs/kit/issues/3176
